src/components/Desktop/Window/Window.svelte

```diff
@@ -112,7 +112,9 @@
   class="container"
   class:dark={$theme.scheme === 'dark'}
   class:active={$activeApp === appID}
-  style="width: {+width / remModifier}rem;height: {+height / remModifier}rem; z-index: {$appZIndices[appID]}"
+  style:width="{+width / remModifier}rem"
+  style:height="{+height / remModifier}rem"
+  style:z-index={$appZIndices[appID] + ''}
```

Comment:
Is this to solve a type issue?

Reply:
yeah. $appZIndices[appID] is a number, and I get a VSCode red squiggle that it should be string

Reply:
Prettier removes the "" here

That's why

Reply:
bruh

Reply:
What? 🤔

Reply:
i understand why prettier does it but still annoying